com.exok.Celeste64.metainfo.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.exok.Celeste64</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <developer_name>Maddy Makes Games Inc., Extremely OK Games, Ltd.</developer_name>
+  <project_group>Maddy Makes Games Inc., Extremely OK Games, Ltd.</project_group>
+  <project_license>MIT</project_license>
+  <name>Celeste 64</name>
+  <summary>Celeste 64: Fragments of the Mountain</summary>
+
+  <url type="homepage">https://exok.com/</url>
+  <url type="help">https://github.com/ExOK/Celeste64</url>
+  <url type="bugtracker">https://github.com/ExOK/Celeste64/issues</url>
+
+  <description>
+    <p>Relive the magic of Celeste Mountain alongside Madeline in this small, heartfelt 3D platformer. Created in a week(ish) by the Celeste team to celebrate the game’s sixth anniversary 🍓✨</p>
+    <p>Controller recommended 🎮</p>
+  </description>
+
+  <launchable type="desktop-id">com.exok.Celeste64.desktop</launchable>
+
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://raw.githubusercontent.com/brunofin/flathub/celeste64/screenshots/1.png</image>
+    </screenshot>
+    <screenshot >
+      <image type="source">https://raw.githubusercontent.com/brunofin/flathub/celeste64/screenshots/2.png</image>
+    </screenshot>
+    <screenshot >
+      <image type="source">https://raw.githubusercontent.com/brunofin/flathub/celeste64/screenshots/3.png</image>
+    </screenshot>
+  </screenshots>
+
+  <categories>
+    <category>Game</category>
+  </categories>
+  
+  <content_rating type="oars-1.0">
+  </content_rating>
```

Comment:
```suggestion
  <content_rating type="oars-1.1" />
```
https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/#oars-information